package/src/components/ChannelList/hooks/usePaginatedChannels.ts

```diff
@@ -88,7 +90,6 @@ export const usePaginatedChannels = <
     filtersRef.current = filters;
     sortRef.current = sort;
     isQueryingRef.current = true;
-    setError(undefined);
```

Comment:
why is this removed? on every query removing the error seems fine to me..

Reply:
if its causing extra rerender.. we have to merge the state somehow

Reply:
There's already a same log on line 136

Reply:
I don't understand why do we need it twice, if you can help here?

Reply:
Its better to remove the line 136 and keep this actually.. 

this one is essential.. removing any error state before making a request, 136 removes any error if request was successful.. both places is also fine, if state is same it wont rerender

Reply:
Removing the 2nd one then